specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/apigateway.json

```diff
@@ -0,0 +1,614 @@
+          "description": "Creation UTC date of the API Management gateway.The date conforms to the following format: `yyyy-MM-ddTHH:mm:ssZ` as specified by the ISO 8601 standard.",
+          "readOnly": true
+        },
+        "frontend": {
+          "$ref": "#/definitions/FrontendConfiguration",
+          "description": "Information regarding how the gateway should be exposed."
+        },
+        "backend": {
+          "$ref": "#/definitions/BackendConfiguration",
+          "description": "Information regarding how the gateway should integrate with backend systems."
+        },
+        "configurationApi": {
+          "$ref": "#/definitions/ConfigurationApi",
+          "description": "Information regarding the Configuration API of the API Management gateway. This is only applicable for API gateway with Standard SKU."
+        }
+      },
+      "description": "Base Properties of an API Management gateway resource description."
+    },
+    "ApiManagementGatewayListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApiManagementGatewayResource"
+          },
+          "description": "Result of the List API Management gateway operation."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Link to the next set of results. Not empty if Value contains incomplete list of API Management services."
+        }
+      },
+      "required": [
+        "value"
+      ],
+      "description": "The response of the List API Management gateway operation."
+    },
+    "ApiVersionConstraint": {
+      "type": "object",
+      "properties": {
+        "minApiVersion": {
+          "type": "string",
+          "description": "Limit control plane API calls to API Management gateway with version equal to or newer than this value.",
+          "example": "2019-01-01"
+        }
+      },
+      "description": "Control Plane Apis version constraint for the API Management gateway."
+    },
+    "ApiManagementGatewaySkuProperties": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of the Sku.",
+          "externalDocs": {
+            "url": "https://azure.microsoft.com/en-us/pricing/details/api-management/"
+          },
+          "enum": [
+            "Standard"
+          ],
+          "x-ms-enum": {
+            "name": "SkuType",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Standard",
+                "description": "Standard SKU of the API gateway."
+              }
+            ]
+          }
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Capacity of the SKU (number of deployed units of the SKU)"
+        }
+      },
+      "required": [
+        "name",
+        "capacity"
```

Comment:
>"capacity" [](http://example.com/codeflow?start=8&length=10)

Based on spec only `Sku.name` is required

[https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/resource-api-reference.md#put-resource](https://github.com/cloud-and-ai-microsoft/resource-provider-contract/blob/master/v1.0/resource-api-reference.md#put-resource)

Reply:
Good point, updated